ansible/playbooks/roles/repository/files/download-requirements/src/command/dnf.py

```diff
@@ -114,6 +122,11 @@ def is_repo_enabled(self, repo: str) -> bool:
 
         return False
 
+    def are_all_repos_enabled(self, repos: List[str]) -> bool:
+        enabled_repos = self.__get_repo_ids()
+
+        return all(repo in enabled_repos for repo in repos)
```

Comment:
return all(repo in self.__get_repo_ids() for repo in repos)

Reply:
Missing brackets [ ]

Reply:
IMO current version has better readability :)